worlds/sc2wol/Options.py

```diff
@@ -103,15 +86,33 @@ class PlayerColor(Choice):
     default = option_default
 
 
-class ShuffleProtoss(DefaultOnToggle):
-    """Determines if the 3 protoss missions are included in the shuffle if Vanilla mission order is not enabled.
-    If turned off, the 3 protoss missions will not appear and Protoss units are removed from the pool."""
-    display_name = "Shuffle Protoss Missions"
+class EnableWolMissions(DefaultOnToggle):
+    """
+    Enables missions from main Wings of Liberty campaign.
+    """
+    display_name = "Enable Wings of Liberty missions"
+
+
+class EnableProphecyMissions(DefaultOnToggle):
+    """
+    Enables missions from Prophecy mini-campaign.
+    """
+    display_name = "Enable Prophecy missions"
+
+
+class ShuffleCampaigns(DefaultOnToggle):
+    """
+    Shuffles the missions between campaigns if enabled.
+    Only available for Vanilla Shuffled and Mini Campaign mission order
+    """
+    display_name = "Shuffle Campaigns"
```

Comment:
Suggestion: rather than going the route of adding a new exclusion option for every new mission group, have one option e.g. `exclude_missions_by_tag` that lets you specify a group / list of groups to exclude all missions from. This could also play nicer with a future `lock_missions` option, which could have the behaviour of overriding missions excluded by group to e.g. exclude all no build except piercing the shroud.

It makes adding new tags easier, too. I took a stab at some initial tags (though underutilized them) in PR #55.
```Python
# MissionTables.py
mission_tags = [
    # ...
    "Welcome to the Jungle":    MissionTags.Covert | MissionTags.VsProtoss,
    "Breakout":                 MissionTags.Covert | MissionTags.VsTerran | MissionTags.NoBuild | MissionTags.Hero,
    "Ghost of a Chance":        MissionTags.Covert | MissionTags.VsTerran | MissionTags.NoBuild | MissionTags.Hero,
    "The Great Train Robbery":  MissionTags.Rebellion | MissionTags.VsTerran,
# ...
]
```

Reply:
Adding a tag list to missions? I think it's an idea for future but outside of scope of this refactoring (Adding an SC2Mission a list attribute of tags[matchup, etc])

Reply:
This option is for Vanilla Shuffled/Mini Campaign to allow to shuffle missions cross campaigns